conda_smithy/templates/travis.yml.tmpl

```diff
@@ -68,9 +63,10 @@ install:
       conda install --yes --quiet conda-forge-build-setup
       {% if build_setup -%}
       {{ build_setup }}{% endif %}
+      conda install -n root --quiet --yes -c defaults conda-build
```

Comment:
Remove this line before merging (version constraint should come from conda-forge-build-setup)